casbin_async_redis_adapter/adapter.py

```diff
@@ -98,6 +98,10 @@ async def _delete_policy_lines(self, ptype, rule):
         else:
             await self.client.lrem(self.key, 0, json.dumps(line.dict()))
 
+    async def _update_policy_line(self, ptype, old_rule, new_rule):
+        # TODO
```

Comment:
Hi, @BustDot 

Can you help with this? 

I'm trying to migrate async-sqlalchemy-adapter to async-redis-adapter, but I'm missing some execution functions. I've added some, but this function is beyond my reach.

Reply:
@wu-clan he may be very busy to help on this. Can you do it with the help of ChatGPT? Or @leeqvip can you help on this?

Reply:
I will try as much as I can.